hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -606,6 +606,13 @@ If the value is not read back, then the requesting interface should wait and try
 When an agent is done with the mutex, it releases the mutex by explicitly writing a 0 to the claim register.
 This resets the mux to select no one and also holds the request interface in reset.
 
+#### Vendor-specific Test Control Register
+
+Certain OTP macros require special configuration bits to be set during the test phases.
+Hence, the life cycle CSRs include register {{< regref "OTP_TEST_CTRL" >}}, which iis reserved for vendor-specific test control bits.
+Its value is only forwarded to the OTP macro in RAW, TEST_* and RMA life cycle states.
+In all other life cycle states, a value of 0 will be sent to the OTP macro.
```

Comment:
Something that occurs to me is to actually check the output is zero when in PROD*/DEV* states and generate an alert if it's not?  Might be overkill, might not be.